# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libchipcard/0001-libchipcard-5.0.0-pcsclite-use-SCARD_READERSTATE.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fixed a bug (use of SCARD_READERSTATE_A is deprecated with current libpcsclite)
Origin: Upstream

Index: trunk/src/lib/client/base/client_p.h
===================================================================
--- trunk/src/lib/client/base/client_p.h	(revision 588)
+++ trunk/src/lib/client/base/client_p.h	(revision 589)
@@ -42,7 +42,7 @@
   SCARDCONTEXT scardContext;
 
   int pnpAvailable;
-  SCARD_READERSTATE_A readerStates[MAX_READERS];
+  SCARD_READERSTATE readerStates[MAX_READERS];
   int readerCount;
   int lastUsedReader;
   LPSTR readerList;
